src/labels/CustomLabels.labels

```diff
@@ -5324,6 +5324,15 @@ This process may take some time, but you can close this page and the process wil
         <value>The custom Relationships fields you do not want synchronized between the two contacts in a relationship pair. Control or command click to select more than one field.</value>
     </labels>
     <labels>
+        <fullName>stgHelpRespectDuplicateRuleSettings</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>stgHelpRespectDuplicateRuleSettings</shortDescription>
```

Comment:
we've been asking that all new labels actually have a useful description for the localizers.  it's very unfortunate that salesforce automatically pastes the label into the description, which led to most of our old labels not having useful text.

Reply:
Good to know. I'll update the label.